.docker/Dockerfile

```diff
@@ -1,36 +1,40 @@
+# syntax=docker/dockerfile:1.7-labs
```

Comment:
This is neat. I hadn't seen this. When we merge, we should add an issue to track when that feature becomes stable, so that we can use the stable syntax

Reply:
It was new to me as well.  I assume it requires a recent version of docker (or docker-compose plugin);  since builds happen in the Github action (or on our desktops), I don't think there are concerns with supporting end users with older Docker installs.

Agreed;  I note there are two newer Docker compose specs (1.8.0 and 1.9.0), but I don't know if this feature has been integrated into those, nor what versions of Docker are required to access those specs.